doc/modules/model_evaluation.rst

```diff
@@ -860,9 +860,9 @@ specified by the ``average`` argument to the
 :func:`fbeta_score`, :func:`precision_recall_fscore_support`,
 :func:`precision_score` and :func:`recall_score` functions, as described
 :ref:`above <average>`. Note that for "micro"-averaging in a multiclass setting
-with all labels included will produce equal precision, recall and :math:`F`,
-while "weighted" averaging may produce an F-score that is not between
-precision and recall.
+with all labels included precision, recall and :math:`F` are all equal, and
```

Comment:
Would there be a comma after included?

Reply:
you're the native speaker ;) I'll reformulate the sentence.